### ISSUE TITLE
Addon not working at all

### ISSUE THREAD

**Issue by TDubsz** [kesava-wow/kuinameplates2#743]
I once updated my profile with the string of a streamer and ever since, the addon doesn´t work AT ALL. i cant even open the settings.

tried reinstalling it severall times, delete all settings thru curseforge, deleted the  folders manually and nothing helps.
![Screenshot 2024-11-16 124159](https://github.com/user-attachments/assets/100fa353-055d-4a2c-bf62-ad5adbfbdce9)

thats the error that´s showing up in WoW.

Using it on Retail.

**Comment by TDubsz** [comment]
I just think it´s weird, that even reinstalling it, deleteing the settings via curseforge, deleting everything manually its not changing anything lol

**Comment by crab-cast** [comment]
Yeah, made me think it might be a conflict with another addon, but I disabled everything except for KNP and it's still borked.

**Comment by Stanzilla** [comment]
Which string did you import?

**Comment by TDubsz** [comment]
https://pastebin.com/Mf2H9bXE

**Comment by crab-cast** [comment]
https://pastebin.com/Mf2H9bXE
https://www.automaticjak.com/exports

I also imported Jak's KNP string. Seems that's the common thread.

Not quite sure how that bricked the addon though. Maybe there's some persistent saved variable that I'm missing to delete for a fresh install?

**Comment by crab-cast** [comment]
Ok I was able to get back to a fresh install by fully closing game, deleting the config files in the following two places:

\World of Warcraft\_retail_\WTF\Account\[Account Name]\[Realm]\[Character Name]\SavedVariables\Kui_Nameplates_Core.lua (+ .bak)
\World of Warcraft\_retail_\WTF\Account\[Account Name]\SavedVariables\Kui_Nameplates_Core.lua (+ .bak)

and then starting back up. When I tried before, looks like they were being rewritten before I could reload.

Still not quite sure what about the string caused the issue, I suspect it was just from an out of date addon version.

**Comment by TDubsz** [comment]
Haha funny, so its the string, tho i had it before, just re copy-pasted it cause i thought it might got updated

**Comment by TDubsz** [comment]
funny, that helped me to

it still says tho, that the jak profile is loaded so idk :D but seems to work!
